### ISSUE TITLE
Doc update for Azure Logic App

### ISSUE THREAD

**Issue by alan0609** [pulumi/pulumi#7268]
The example for [creating a workflow with c#](https://www.pulumi.com/docs/reference/pkg/azure-native/logic/workflow/#definition_csharp) doesn't seems to work.

As far as I can tell the line 

`Definition =`

should be 

`Definition = new Dictionary<string, object>`

**Comment by alan0609** [comment]
Looks like there is a bit more to it than that as well.  To create a definition for a "blank" logic app I had to use

```
Definition = new Dictionary<string, object>
{
    { "$schema", "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#" },
    { "contentVersion", "1.0.0.0" },
    { "actions",  new Dictionary<string, object>{} },
    { "triggers", new Dictionary<string, object>{} },
    { "parameters", new Dictionary<string, object>{} },
    { "outputs", new Dictionary<string, object>{} }
};
```

**Comment by infin8x** [comment]
@alan0609 not sure what's going on here; the code in the example appears to be correct C#. Can you share a screenshot of the error you're seeing when you use the example code?

**Comment by alan0609** [comment]
Hi @infin8x , sorry for the long delay in getting back to you.  If I do a pulumi new azure-csharp and overwrite the code in MyStack.cs with the code from that example I end up with 140 errors :).  Are you saying that it works for you?

Cheers

Alan

**Comment by emiliza** [comment]
Yep - that documentation is incorrect and needs to be updated. It should look a little more like this example: https://github.com/pulumi/examples/blob/master/azure-cs-cosmosdb-logicapp/MyStack.cs#L113-L203

We'll need to update that documentation block to include `new Dictionary<string, object>` in all the appropriate places
